### PR TITLE
Add shopping use case

### PR DESCRIPTION
**Pull request by lesters**
@jobergum PTAL.
@ldalves FYI.

First iteration on this.

### CODE REVIEW

**Comment by lesters** [comment]
@kkraune FYI.

File: use-case-shopping/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<pre data-test="exec">
+$ docker exec vespa bash -c 'java -jar /opt/vespa/lib/jars/vespa-http-client-jar-with-dependencies.jar \
+    --file /vespa-sample-apps/use-case-shopping/feed_items.json --host localhost --port 8080'
+</pre>
+
+Feed reviews:
+
+<pre data-test="exec">
+$ docker exec vespa bash -c 'java -jar /opt/vespa/lib/jars/vespa-http-client-jar-with-dependencies.jar \
+    --file /vespa-sample-apps/use-case-shopping/feed_reviews.json --host localhost --port 8080'
```
Comment (jobergum) [inline use-case-shopping/README.md]: The review json has several fields with control characters which we don't allow which causes feed failures which gives a bad user experience when feeding the data set.

File: use-case-shopping/src/main/application/searchdefinitions/item.sd [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            indexing: summary | attribute
+            summary-to: short
+        }
+
+        field timestamp type long {
+            indexing: summary | attribute
+        }
+
+        field brand type string {
+            indexing: summary | attribute | index
```
Comment (jobergum) [inline use-case-shopping/src/main/application/searchdefinitions/item.sd]: Should brand be match:word/exact?

File: use-case-shopping/README.md (hunk 1 above)
Comment (lesters) [reply use-case-shopping/README.md]: Quick fix: filtered in conversion script.

File: use-case-shopping/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Please refer to [Vespa use cases](https://docs.vespa.ai/documentation/use_cases.html) 
+for more information.
+
+Included scripts to convert data from Julian McAuley's Amazon product data set
+(http://jmcauley.ucsd.edu/data/amazon/links.html)
+to a Vespa data feed. This repository contains a small sample of this data from 
+the sports and outdoor category, but you can download other data from the site 
+above and use the scripts to convert. 
+
+### How to run
```
Comment (kkraune) [inline use-case-shopping/README.md]: Add what is required to run the sampleapp, like python 3

Comment (kkraune) [reply use-case-shopping/README.md]: or, we could pack python3 with the image and run the processing inside the container

**Comment by jobergum** [comment]
Agree with @kkraune on the script dependencies to move this into the docker container. The centos based image already comes with python2.7

**Comment by lesters** [comment]
Removed requirement for Python3 - the conversion script works on both 2 and 3 .

**Comment by ldalves** [comment]
💯

File: use-case-shopping/src/main/application/searchdefinitions/item.sd (hunk 2 above)
Comment (lesters) [reply use-case-shopping/src/main/application/searchdefinitions/item.sd]: Brand is now attribute only.

**Pull request merged**